README.md

```diff
@@ -15,22 +16,24 @@
 
 ---
 
-Kubernetes is wonderful, but requires scarce time and expertise to set up and manage.
-Other solutions demand you have already set up Linux correctly, have all your servers in the same network, and meet their other criteria.
-And hybrid clusters that span cloud and data center?
-Good luck with that!
-
-Omni allows you to start with bare metal, virtual machines or a cloud provider, and create clusters spanning all of your locations, with a few clicks.
+Omni manages Kubernetes on bare metal, virtual machines, or in a cloud.
+Built on [Talos Linux](https://talos.dev) by the folks at [Sidero](https://siderolabs.com).
 
-You provide the machines – edge compute, bare metal, VMs, or in your cloud account. Boot from an Omni image. Click to allocate to a cluster. That’s it!
+Boot from an Omni image. Click to allocate to a cluster. That’s it!
 
 - Vanilla Kubernetes, on your machines, under your control.
 - Elegant UI for management and operations
 - Security taken care of – ties into your Enterprise ID provider
```

Comment:
Should we remove whitespaces here for consistency?

Reply:
yep, sorry. I pushed that to my fork by accident instead of here.